conn.go

```diff
@@ -38,12 +38,12 @@ type ConnOptions struct {
 	// Open frame and TLS ServerName (if not otherwise set).
 	HostName string
 
-	// IdleTimeout specifies the maximum period in milliseconds between
+	// IdleTimeout specifies the maximum period in nanoseconds between
```

Comment:
I think we could simply remove the "in <units>" part of the text as it should be implied that that maximum period is any valid `time.Duration` value.